public/app/features/dashboard-scene/panel-edit/PanelOptionsPane.tsx

```diff
@@ -21,14 +42,86 @@ export class PanelOptionsPane extends SceneObjectBase<PanelOptionsPaneState> {
   static Component = ({ model }: SceneComponentProps<PanelOptionsPane>) => {
     const { panelManager } = model;
     const { panel } = panelManager.state;
-    const { title } = panel.useState();
+    const { title, description, displayMode, pluginId } = panel.useState();
     const styles = useStyles2(getStyles);
+    const [isVizPickerOpen, setVizPickerOpen] = useState(true);
 
     return (
-      <div className={styles.box}>
-        <Field label="Title">
-          <Input value={title} onChange={(evt) => panel.setState({ title: evt.currentTarget.value })} />
-        </Field>
+      <div className={styles.wrapper}>
+        {!isVizPickerOpen && (
+          <VisualizationButton
+            pluginId={pluginId}
+            onClick={() => {
+              setVizPickerOpen(true);
+            }}
+          />
+        )}
+        <div className={styles.box}>
+          {isVizPickerOpen && (
+            <PanelVizTypePicker panelManager={panelManager} onChange={() => setVizPickerOpen(false)} />
+          )}
+          {!isVizPickerOpen && (
```

Comment:
Think we may skip this part for now and just render options placegolder before we migrate the actual options forms, wdyt?

Reply:
yea, these needs to be done declaratively anyway (OptionsPaneItemDescriptor) to support the search